### PR TITLE
fix flutter analyze errors

### PR DESCRIPTION
**Pull request by pearlson16**
# Description
Went through and handled all the code errors until flutter analyze passes

Fixes Jira Ticket # [AX-374](https://athletex.atlassian.net/browse/AX-374) [Frontend] Fix all Flutter Analyze issues

## Type of change
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

image

# How Has This Been Tested?
I ran the the app and checked each page and made sure there are no errors

### CODE REVIEW

**Review by ketonkss4 — APPROVED** [review]

File: lib/service/Controller/Scout/LSPController.dart [hunk 1]
```diff
@@ -46,24 +43,14 @@ class LSPController extends GetxController {
...
 
   Future<void> approve() async {
-    BigInt amount =  BigInt.from(createAmt.value) * BigInt.parse('250000000000000000');
     print("[Console] Inside approve()");
-    EthereumAddress address =
-        EthereumAddress.fromHex("0x76d9a6e4cdefc840a47069b71824ad8ff4819e85");
-    final tokenClient =
-        Web3Client("https://matic-mumbai.chainstacklabs.com", Client());
-    Erc20 axt = Erc20(address: address, client: tokenClient);
     try {
```
Comment (user2745) [inline lib/service/Controller/Scout/LSPController.dart:47]: what's the reason this was deleted?

File: lib/service/Controller/Swap/SwapController.dart [hunk 2]
```diff
@@ -121,9 +115,8 @@ class SwapController extends GetxController {
 
   Future<void> swapFromAX() async {
     EthereumAddress tknA = EthereumAddress.fromHex("$address1.value");
-    BigInt amountIn = BigInt.from(amount1.value);
 
-    List<EthereumAddress> path = [tknA, AXTAddress];
+    List<EthereumAddress> path = [tknA, atxAddress];
```
Comment (user2745) [inline lib/service/Controller/Swap/SwapController.dart:119]: axtAddress

File: lib/service/Controller/Swap/SwapController.dart [hunk 3]
```diff
@@ -112,7 +106,7 @@ class SwapController extends GetxController {
     EthereumAddress tknA = EthereumAddress.fromHex("$address1.value");
     BigInt amountIn = BigInt.from(amount1.value);
     EthereumAddress to = await controller.credentials.extractAddress();
-    List<EthereumAddress> path = [tknA, AXTAddress];
+    List<EthereumAddress> path = [tknA, atxAddress];
```
Comment (user2745) [inline lib/service/Controller/Swap/SwapController.dart:109]: axtAddress

File: lib/service/Controller/Swap/SwapController.dart [hunk 4]
```diff
@@ -21,7 +19,7 @@ class SwapController extends GetxController {
       EthereumAddress.fromHex("0xe06DC83e310807BAcF2e1776925bC19Fa3659D78");
   final EthereumAddress routerAddress =
       EthereumAddress.fromHex("0x7EFc361e568d0038cfB200dF9d9Be27943e19017");
-  final EthereumAddress AXTAddress =
+  final EthereumAddress atxAddress =
```
Comment (user2745) [inline lib/service/Controller/Swap/SwapController.dart:22]: axtAddress

**Review by user2745 — CHANGES_REQUESTED** [review]
Some extra comments: 

Prices.dart can be totally deleted
Dialog seems to have state management totally removed

File: lib/service/Controller/Swap/SwapController.dart (hunk 4 above)
Comment (pearlson16) [reply lib/service/Controller/Swap/SwapController.dart:22]: corrected this

File: lib/service/Controller/Swap/SwapController.dart (hunk 3 above)
Comment (pearlson16) [reply lib/service/Controller/Swap/SwapController.dart:109]: this is corrected

File: lib/service/Controller/Swap/SwapController.dart (hunk 2 above)
Comment (pearlson16) [reply lib/service/Controller/Swap/SwapController.dart:119]: this is corrected

File: lib/service/Controller/Scout/LSPController.dart (hunk 1 above)
Comment (pearlson16) [reply lib/service/Controller/Scout/LSPController.dart:54]: i readded this and some will continue this work

Comment (user2745) [reply lib/service/Controller/Scout/LSPController.dart:54]: thank you

**Comment by user2745** [comment]
This is approved

**Review by user2745 — APPROVED** [review]

**Pull request merged**